package.json

```diff
@@ -29,7 +29,7 @@
   },
   "devDependencies": {
     "babel": "^5.8.21",
-    "babel-eslint": "^4.1.3",
+    "babel-eslint": "^4.1.8",
```

Comment:
The tests all seem to pass on travis but break on circle. So it is definitely a circle-ci issue, maybe with the xvfb implementation.